Source/WebKit/UIProcess/ProvisionalFrameProxy.cpp

```diff
@@ -49,7 +49,7 @@
 
 namespace WebKit {
 
-ProvisionalFrameProxy::ProvisionalFrameProxy(WebFrameProxy& frame, Ref<WebProcessProxy>&& process, const WebCore::ResourceRequest& request, bool newProcess)
+ProvisionalFrameProxy::ProvisionalFrameProxy(WebFrameProxy& frame, Ref<WebProcessProxy>&& process, const WebCore::ResourceRequest& request, bool shouldUseNewProcess)
```

Comment:
I know I suggested this name but after I read the rest of the patch, I realized that my understanding was likely incorrect. This isn't a `shouldUseNewProcess` but rather a `isNewProcess` I believe.

Reply:
or didCreateNewProcess.